vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -56,21 +80,21 @@ the task list for that page.
  
 The default PAM implementation exposes the top level task fields in the task list for selection. It is possible to fetch task and process variables from the task list for rendering, but this is disabled by default to optimize performance.
```

Comment:
to fetch the task and process variables

Reply:
"the" is not necessary, and begs the question of *which* variables "the" refers to. omitting "the" leaves it ambiguous. i don't know enough to resolve the statement further (e.g. all are fetched? or just some? is this editable?)